src/Filesystem.cpp

```diff
@@ -29,70 +29,33 @@
 using namespace NAS2D;
 using namespace NAS2D::Exception;
 
-//enum MountPosition
-//{
-//	MOUNT_PREPEND = 0,
-//	MOUNT_APPEND = 1,
-//};
-
-/**
- * Default c'tor.
- */
-//Filesystem::Filesystem() noexcept
-//	: mVerbose(false)
-//{}
-
-/**
- * Shuts down PhysFS and cleans up.
- */
-//Filesystem::~Filesystem() noexcept
-//{
-//	if (PHYSFS_isInit()) { PHYSFS_deinit(); }
-//}
 
 /**
  * Shuts down PhysFS and cleans up.
  */
 void Filesystem::init(const std::string& /*argv_0*/, const std::string& /*appName*/, const std::string& /*organizationName*/, const std::string& dataPath) noexcept
 {
-	if (mIsInit)
-	{
-		return;
-	}
-	//if (PHYSFS_isInit()) { throw filesystem_already_initialized(); }
-
-	//if (PHYSFS_init(argv_0.c_str()) == 0)
-	//{
-	//	throw filesystem_backend_init_failure(getLastPhysfsError());
-	//}
-
-	//if (PHYSFS_setSaneConfig(organizationName.c_str(), appName.c_str(), nullptr, false, false) == 0)
-	//{
-	//	std::cout << std::endl << "(FSYS) Error setting sane config. " << getLastPhysfsError() << "." << std::endl;
-	//}
+	if (mIsInit) { return; }
 
 	namespace FS = std::filesystem;
 	FS::path p = dataPath;
 	std::error_code ec{};
 	p = FS::canonical(p, ec);
 	if (ec)
 	{
-		std::cerr << "Filesystem::init failed:\n"
-				  << "dataPath: " << p
-				  << "\nReasons for failure:\n"
+		std::cerr << "Filesystem::init failed:" << std::endl
+				  << "dataPath: " << p << std::endl
+				  << "Reasons for failure:" << std::endl
```

Comment:
`std::cerr` automatically flushes the stream, so does `std::endl`. Either the `std::endl` is unnecessary or switch to using the, incorrect in this context, `std::cout`.

Ref: https://www.youtube.com/watch?v=GMqQOEZYVJQ
Ref: https://www.youtube.com/watch?v=lHGR_kH0PNA

Reply:
In this context the intention is to keep the code consistent with the rest of the code base in terms of line endings. While it may be unnecessary as two flushes are being done the consistency improvement is worth the change.

Reply:
Or implement #280, and remove this code as no longer relevant. :wink:

Reply:
Sounds good. :)